zuul-core/src/main/java/com/netflix/zuul/netty/server/http2/Http2OrHttpHandler.java

```diff
@@ -44,7 +43,6 @@
  */
 public class Http2OrHttpHandler extends ApplicationProtocolNegotiationHandler {
     public static final AttributeKey<String> PROTOCOL_NAME = AttributeKey.valueOf("protocol_name");
-    public static final AttributeKey<Http2Connection> H2_CONN_KEY = AttributeKey.newInstance("h2_connection");
```

Comment:
What happened to the `H2_CONN_KEY` attribute?

Reply:
It was never used internally or externally, so I removed it.   The connection is not accessible in the new API as well.